api/envoy/config/common/accesslog/common.proto

```diff
@@ -0,0 +1,49 @@
+syntax = "proto3";
```

Comment:
@mattklein123 , @PiotrSikora, @mandarjog , @kyessenov  : what's the best path to put this in? My current path fails with versions error..
How about in config/accesslog/v3 itself? (This would be used in v2 route proto too)

Reply:
Everything should just go into v3 packages. We don't allow unversioned packages or changes to v2 protos anymore).

Reply:
@htuch : so I should not edit v2 protos too? In my current example I add v3.ResponseFlagFilter in v2.route_components and get a lot of doc errors.. Looks like docs are structured to not refer a v3 proto inside of v2 proto? 

Should I not make a change to v2 proto at all and just go ahead and make changes to v3 protos only? Would there be compatibility issue because of this? say a proxy using v2 configs?

Reply:
Yes, v2 is frozen, older clients can't consume these new features.

Reply:
Thanks @htuch , I will remove changes to v2 api in that case completely...

Reply:
Fixed..PTAL